### PR TITLE
feat: add consumer migration tool

### PR DESCRIPTION
**Pull request by guseggert**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -56,18 +56,20 @@ require (
 	github.com/prometheus/client_golang v1.14.0
 	github.com/samber/lo v1.36.0
 	github.com/stretchr/testify v1.8.1
+	github.com/urfave/cli/v2 v2.25.0
```
Comment (Jorropo) [inline go.mod]: Let's make it it's own module.

File: cmd/migrate/internal/migrator.go [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+	}
+
+	if !fileChanged {
+		return nil
+	}
+
+	f, err := os.Create(filePath)
+	if err != nil {
+		return err
+	}
```
Comment (Jorropo) [inline cmd/migrate/internal/migrator.go:76]:
```suggestion
	}
	defer f.Close()
```

File: cmd/migrate/internal/migrator_test.go [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+		{
+			name: "happy case",
+		},
+	}
+	for _, c := range cases {
+		t.Run(c.name, func(t *testing.T) {
+
+		})
+	}
+}
```
Comment (Jorropo) [inline cmd/migrate/internal/migrator_test.go]: ?

**Review by Jorropo — CHANGES_REQUESTED** [review]

File: cmd/migrate/internal/migrator_test.go (hunk 3 above)
Comment (guseggert) [reply cmd/migrate/internal/migrator_test.go]: removed

**Review by Jorropo — APPROVED** [review]

**Pull request merged**